greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExportState.java

```diff
@@ -0,0 +1,7 @@
+package com.greenback.kit.model;
+
+public enum AutoExportState {
```

Comment:
This is the state of the auto export -- not if its running.  Active, archived, or deleted would be the only states I think we'd have.